packages/agoric-cli/lib/open.js

```diff
@@ -102,9 +102,6 @@ export default async function walletMain(progname, rawArgs, powers, opts) {
   process.stdout.write(`${walletUrl}\n`);
   if (opts.browser) {
     const browser = opener(walletUrl);
-    browser.unref();
-    process.stdout.unref();
-    process.stderr.unref();
-    process.stdin.unref();
+    await new Promise(resolve => browser.on('exit', resolve));
```

Comment:
How does this relate to Windows? Why wasn't this necessary on linux / mac?

Reply:
It's a race on Windows; it terminates the opener process if the `agoric open` exits too soon (which it nearly always did in my testing).

Reply:
ah.

worth a comment? or a note in the commit message?

Reply:
Done.